src/utils/sentry.ts

```diff
@@ -7,22 +7,27 @@ let sentryInitialized = false;
 
 function getAppDetails() {
   const pathName = window.location.pathname.split('/');
-  let appGroup;
+  let appGroup = 'insights';
   let appName;
-  let betaCheck;
-
+  let environment = 'stable';
   if (pathName[1] === 'beta') {
-    betaCheck = ' Beta';
     appGroup = pathName[2];
-    appName = appGroup === 'landing' ? 'landing' : pathName[3];
+    appName = pathName[3];
+    environment = 'beta';
   } else {
-    betaCheck = '';
     appGroup = pathName[1];
-    appName = appGroup === 'landing' ? 'landing' : pathName[2];
+    if (appGroup === 'openshift') {
```

Comment:
@adonispuente we could add a test to exercise these conditionals/functions as we have regressed this without realizing in the past (for example the `beta` logic not working any longer).